src/applications/facility-locator/config.js

```diff
@@ -34,54 +34,43 @@ export const resolveParamsWithUrl = (
   bounds,
   center,
   radius,
+  allUrgentCare = false,
 ) => {
   const filterableLocations = ['health', 'benefits', 'provider'];
   let facility;
   let service;
   let url = api.url;
-  let perPage = 20;
   let roundRadius;
+  let perPage = 20;
 
   switch (locationType) {
     case 'urgent_care':
       if (serviceType === 'UrgentCare') {
         facility = 'health';
         service = 'UrgentCare';
-      } else if (serviceType === 'NonVAUrgentCare') {
+      } else if (serviceType === 'NonVAUrgentCare' && allUrgentCare) {
         facility = 'urgent_care';
         url = api.ccUrl;
-      } else {
-        // MashUp coming up
-        url = api.allUrgentCareUrl;
+        perPage = 40;
+      } else if (serviceType === 'NonVAUrgentCare' && !allUrgentCare) {
+        facility = 'urgent_care';
+        url = api.ccUrl;
+        perPage = 20;
       }
       break;
     case 'pharmacy':
     case 'provider':
       facility = locationType;
       service = serviceType;
       url = api.ccUrl;
-      perPage = 10; // because the PPMS back end requires a separate request for each facility
+      perPage = 10;
```

Comment:
Since this comment is no longer relevant, should we bump up `perPage` to 20 here?

Reply:
sounds good to me